### DISCUSSION TITLE
New installation and I don't know what is wrong

### DISCUSSION THREAD

**Discussion by mchyla7** [photoprism/photoprism#1793] [Q&A] [answered]
I have a problem with my new installation, installing on raspberry pi 4 I just install all that I need, install MySQL with a password which I know I make databases and user with access to the database and I still don't know what I make wrong, I start the installation and I'm done at the database with this Error code. `level=fatal msg="dial tcp: lookup mariadb on 127.0.0.11:53: no such host`. I can connect to Database but I don't understand this error. Someone help?

Log:
Recreating photoprism_mariadb_1 ... done
Recreating photoprism_photoprism_1 ... done
Attaching to photoprism_mariadb_1, photoprism_photoprism_1
photoprism_1  | started as root
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
photoprism_1  | running as root
photoprism_1  | photoprism start
photoprism_mariadb_1 exited with code 1
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
photoprism_mariadb_1 exited with code 1
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
photoprism_mariadb_1 exited with code 1
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
photoprism_1  | time="2021-12-06T19:29:52Z" level=info msg="Help us make a difference and become a sponsor today!"
photoprism_1  | time="2021-12-06T19:29:52Z" level=info msg="Visit https://docs.photoprism.org/funding/ to learn more."
photoprism_1  | time="2021-12-06T19:29:52Z" level=info msg="config: make sure your server has enough swap configured to prevent restarts when there are memory usage spikes"
photoprism_mariadb_1 exited with code 1
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
photoprism_mariadb_1 exited with code 1
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
photoprism_mariadb_1 exited with code 1
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
photoprism_mariadb_1 exited with code 1
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
mariadb_1     | standard_init_linux.go:228: exec user process caused: exec format error
**photoprism_1  | time="2021-12-06T19:30:59Z" level=fatal msg="dial tcp: lookup mariadb on 127.0.0.11:53: no such host"
photoprism_photoprism_1 exited with code 0**

**Comment by lastzero** [comment, marked as answer]
Enable 64 bit Mode as explained in our docs or use the ARMv7 config:

https://dl.photoprism.org/docker/armv7/docker-compose.yml
